embedded-graphics/src/primitives/triangle.rs

```diff
@@ -146,15 +123,15 @@ where
     }
 }
 
-fn sort_two_yx(p1: Coord, p2: Coord) -> (Coord, Coord) {
-    if p1[1] < p2[1] || (p1[1] == p2[1] && p1[0] < p2[0]) {
+fn sort_two_yx(p1: Point, p2: Point) -> (Point, Point) {
```

Comment:
Does/should `Point` and `Size` impl `Ord` and/or `PartialOrd`, etc?

Reply:
They don't and probably shouldn't because there are multiple ways to order 2D points.